lib/vintage_net_qmi/connection.ex

```diff
@@ -92,7 +92,8 @@ defmodule VintageNetQMI.Connection do
 
   defp try_run_configuration(:radio_technologies_set, state) do
     NetworkAccess.set_system_selection_preference(state.qmi,
-      mode_preference: state.radio_technologies
+      mode_preference: state.radio_technologies,
+      roaming_preference: :any
```

Comment:
Is the `roaming_preference` setting here important? Or do you need to set it here for `roaming_disallowed: false` to work?

Reply:
Correct, we could move it be at the provider level but the requires a bit more code changes and I assumed setting it to any does not hurt anything has the modem specific configuration will control if the modem connects to roaming or not. Does that seem like an okay assumption? I don't moving things around a little more to have this conditioned on the service provider configuration.

Reply:
I suppose my question was more clarification. If you were to delete this line would that break super sims from working on BG96's.

Reply:
I can test to verify, I did go in assuming that this was necessary, but it might not be.

Reply:
I don't this anything in the QMI spec on what the default is. I am curious on testing it. At the very least, I think that it's worth a comment explaining why the roaming preference is being hardcoded here.

Reply:
After testing, this line does not look necessary so I am going to delete it now. We can always add this back in if we learn more later on down the road.